provider/cluster_rosa_classic_resource.go

```diff
@@ -629,6 +643,17 @@ func createClassicClusterObject(ctx context.Context,
 			)
 			return nil, errors.New(errHeadline + "\n" + errDecription)
 		}
+		vBuilder := cmv1.NewVersion()
```

Comment:
I'm not sure - but maybe we need to validate that the selected version is compatible with the channel group.

Reply:
Unless I'm missing it, we don't have access to the api server from this function... I actually wanted to do the validation here, but couldn't.
However, `createClassicClusterObject` is only called from Create(), right after the verification of account roles... and the account role verification flow is where we check the version/channel. In effect, it does get checked, just not right here.

Reply:
you're right
the validations are done in the function "Create"- https://github.com/terraform-redhat/terraform-provider-ocm/blob/8c32bac577336a1d2ca868740c72656000b4bda3/provider/cluster_rosa_classic_resource.go#L903

Reply:
That it's because the UTs are running against the `createClassicClusterObject` function